concourse/tasks/gcloud-package-operation.yaml

```diff
@@ -17,4 +17,13 @@ run:
   args:
   - -exc
   - |
-    gcloud pubsub topics publish $TOPIC --message "{\"type\": \"${TYPE}\", \"request\": {\"bucket\": \"gcp-guest-package-uploads\", \"object\": \"((package_path))\", \"universe\": \"((universe))\", \"repo\": \"((repo))\"}}"
+    paths=$(echo "((package_path))" | tr ";" "\n")
+    for path in $paths
+    do
+      if [[ "${objects}" != "" ]]; then
+        objects="${objects}, "
+      fi
+      objects="${objects}{\"bucket\":\"gcp-guest-package-uploads\",\"object\":\"${path}\"}"
+    done
```

Comment:
can't we just write the input to match what this task expects? this is overly complicated

Reply:
done